docs/api/utils/useProxy.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: 'useProxy'
```

Comment:
It's a very handy util, but we are still not sure how we should suggest the usage. Our primary hook is `useSnapshot` and `useProxy` is a wrapper around it.